src/qililab/result/qblox_results/qblox_result.py

```diff
@@ -156,12 +160,18 @@ def samples(self) -> np.ndarray:
         Returns:
             np.ndarray: An array containing the measured samples (0 or 1).
         """
+        if sum(result["measurement"] for result in self.qblox_raw_results) != 0:
+            raise NotImplementedError("Samples for multiple measurements on a single qubit are not supported")
         return self.qblox_bins_acquisitions.samples()
 
     @property
     def array(self) -> np.ndarray:
```

Comment:
Can we add a docstring here explaining the shape of the returned array, and the cases that an error is raised?